server/copy.go

```diff
@@ -134,7 +138,7 @@ func copyFromFileStore(dst, src string, b filestore.FileBackend) error {
 
 func compressDirectory(sourceDir, compressedFile string) error {
 	// Create a new archive file
-	newZipFile, cErr := os.Create(compressedFile)
+	newZipFile, cErr := os.OpenFile(compressedFile, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0750)
```

Comment:
os.Create gives 0660 permissions. 

Why are we changing it giving the user and the group execute permissions? afaik there is nothing to execute. 

Similar to the above, we'd like to restrict the group as only the mattermost process should need to have access

Reply:
Right, I probably misinterpreted what we discussed. Changed to more restrictive, please check.